client/my-sites/post-selector/selector.jsx

```diff
@@ -131,6 +134,7 @@ const PostSelectorPosts = React.createClass( {
 		const checked = this.props.selected === item.ID;
 		const inputType = this.props.multiple ? 'checkbox' : 'radio';
 		const domId = camelCase( this.props.analyticsPrefix ) + '-option-' + itemId;
+		const postType = get( this.props.postTypes, [ item.type, 'labels', 'singular_name' ] );
```

Comment:
>These labels are translated API-side correct?

Actually not, now that you mention it. Localization is opt-in, so [the current logic](https://github.com/Automattic/wp-calypso/blob/bbabbe3ac1df060292bbc8aa3d115640c3b70777/client/lib/post-types-list/list.js#L54-L65) is not localized either. Furthermore, there was a bug in the REST API which was causing default post types (post, page, etc) to not respect the `locale` parameter, which has been fixed in the past day (r131176-wpcom). #3348 seeks to address the need for broad localization support for all `wpcom.js` usage, at which point we can look to use it in the `state/post-types/actions.js` action creator for requesting post types.